src/Math-Core/PMVector.class.st

```diff
@@ -237,6 +237,26 @@ PMVector >> productWithVector: aVector [
 			into: [ :sum :each | n := n + 1. (aVector at: n) * each + sum]
 ]
 
+{ #category : #'as yet unclassified' }
+PMVector >> resizeWithRows: numOfRows Cols: numOfCols [
+	
+	| computedRows rowNum colNum |
+	
+	numOfRows isNil ifFalse: [rowNum := numOfRows ].
+	numOfCols isNil ifFalse: [colNum := numOfCols ].
+	
+	(rowNum isNil and: colNum isNil) ifFalse: [ 
+		numOfRows isNil ifTrue: [ rowNum := self size / colNum ] .
+		numOfCols isNil ifTrue: [ colNum := self size / rowNum ] .
+		].
+	
+	self assert: (rowNum * colNum = self size) description: 'Number of Rows not compatible with number of Columns'.
```

Comment:
I think 'rows' and 'columns' rather than 'Rows'/'Columns' as in this case they're not proper nouns.